content/articles/working-with-system-information-in-c#-windows-form-application/index.md

```diff
@@ -0,0 +1,227 @@
+USe Four `TextBox` controls for display, four `labels`, and two `buttons`, one for checking hardware information and the other for exiting, as shown below.
+
+![Management namespace Design page](/engineering-education/working-with-system-information-in-a-windows-form-c-sharp-program/managementnamespace.png)
+
+#### Step 2: Naming TextBox controls
+As I previously stated, naming conventions must be followed, and textBoxes should be named differently. Textboxes are named `txtProcessorId`, `txtBIOSMaker`, `txtPhysicalMemory`, and `motherboard` in my case.
+
+#### Step 3: Adding the code
+This is the final step, in which we should offer the code that will assist in getting hardware information, which will be coded within a function of the two buttons as follows:
+
+##### Button 1: Check the system information
+Double-click the system information button and add the code below.
+
+```C#
+private void button5_Click(object sender, EventArgs e)
+{
+    ManagementClass management = new ManagementClass("Win32_Processor");
+    ManagementObjectCollection managementobject = management.GetInstances();
+
+    foreach (ManagementObject mngObject in managementobject)
+    {
+        txtProcessorId.Text= mngObject.Properties["ProcessorId"].Value.ToString();
+        break;
+    }
+
+    // Check BIOS cheker
+    ManagementClass management2 = new ManagementClass("Win32_BIOS");
+    ManagementObjectCollection managementobject2 = management2.GetInstances();
+    foreach (ManagementObject mngObject2 in managementobject2)
+    {
+        txtBIOSMaker.Text= mngObject2.Properties["SerialNumber"].Value.ToString();
+        break;
+    }
+
+    // Check physical memory serial number
+    ManagementClass management3 = new ManagementClass("Win32_PhysicalMedia");
+
+    ManagementObjectCollection managementobject3 = management3.GetInstances();
+    foreach (ManagementObject mngObject3 in managementobject3)
+    {
+        txtPhaysicalMemory.Text= mngObject3.Properties["SerialNumber"].Value.ToString();
+        break;
+    }
+
+
+    // Check Motherboard
+    ManagementClass management4 = new ManagementClass("Win32_BaseBoard");
+    ManagementObjectCollection managementobject4 = management4.GetInstances();
+
+    foreach (ManagementObject mngObject4 in managementobject4)
+    {
+        txtMother.Text = mngObject4.Properties["SerialNumber"].Value.ToString();
+        break;
+    }
+}
+```
+
+The `ManagementClass` is used to examine hardware information such as Processor id, BIOS Maker Serial Number, Physical Memory Serial Number, and Motherboard Serial Number in our case, as shown in the code sample above.
+
+To access the `ManagementClass` properties, we must first build the `ManagementClass` class object as well as the `ManagementObjectCollection` class object.
+
+The value attributes provided by the above classes assist us in displaying the value of specific installed hardware.
+
+##### Button 2: Exit button
+Double-click the exit button and add the code below.
+

```

Comment:
This whole section is a duplicate of another section from part 1 in your article. Remove this section and mention that the button's implementation doesn't change.